cmd/crictl/sandbox.go

```diff
@@ -357,15 +373,16 @@ func ListPodSandboxes(client pb.RuntimeServiceClient, opts listOptions) error {
 
 	w := tabwriter.NewWriter(os.Stdout, 20, 1, 3, ' ', 0)
 	if !opts.verbose && !opts.quiet {
-		fmt.Fprintln(w, "SANDBOX ID\tNAME\tSTATE")
+		fmt.Fprintln(w, "SANDBOX ID\tNAME\tNAMESPACE\tSTATE")
```

Comment:
Sorry, I found that for containers we put name at the end. Let's keep this consitent.

Let's do the same `sandbox id\tstate\tname\tnamespace`.

Reply:
OK, will do.

Reply:
Done.